c++/src/Statistics.hh

```diff
@@ -605,6 +676,43 @@ namespace orc {
     }
   };
 
+  class StripeStatisticsImpl: public StripeStatistics {
+  private:
+    std::unique_ptr<StatisticsImpl> columnStats;
+    std::vector<std::vector<std::unique_ptr<const ColumnStatistics> > > rowIndexStats;
```

Comment:
Similar to my previous comment in ORC-87. If we need to be compatible with older C++ standards, this line may not work.